tester/src/test_executors/auxboard.rs

```diff
@@ -95,15 +100,18 @@ impl TestExecutor for AuxBoardTestExecutor {
 			}
 		}
 
+		thread::sleep(time::Duration::from_millis(500));
+		*/
+
 		{
 			let mut l = self.logger.lock().unwrap();
-			l.in_progress("Enabling rotation vector...");
+			l.in_progress("Enabling game rotation vector...");
 		}
 		let start = chrono::Utc::now();
-		match self.bno.enable_rotation_vector(5) {
+		match self.bno.enable_rotation_vector(5) { // TODO enable game rotation vector
```

Comment:
The arduino firmware uses 10ms, not 5

```suggestion
		match self.bno.enable_rotation_vector(10) {
```